ocean_lib/ocean/test/test_ocean_assets.py

```diff
@@ -225,3 +228,66 @@ def test_create_asset_with_address(publisher_ocean_instance):
     assert ocn.assets.create(
         asset.metadata, alice, [auth_service], data_token_address=token.address
     )
+
+
+def test_create_asset_with_owner_address(publisher_ocean_instance):
+    """Tests creation of the asset which has already an owner address."""
+    ocn = publisher_ocean_instance
+    alice = get_publisher_wallet()
+
+    sample_ddo_path = get_resource_path("ddo", "ddo_sa_sample.json")
+    asset = DDO(json_filename=sample_ddo_path)
+    asset.metadata["main"]["files"][0]["checksum"] = str(uuid.uuid4())
+    my_secret_store = "http://myownsecretstore.com"
+    auth_service = ServiceDescriptor.authorization_service_descriptor(my_secret_store)
+
+    token = ocn.create_data_token(
+        "DataToken1", "DT1", from_wallet=alice, blob="foo_blob"
+    )
+
+    assert ocn.assets.create(
+        asset.metadata,
+        alice,
+        [auth_service],
+        owner_address=alice.address,
+        data_token_address=token.address,
+    )
+
+
+def test_create_asset_without_dt_address(publisher_ocean_instance):
+    """Tests creation of the asset which has not the data token address."""
+    ocn = publisher_ocean_instance
+    alice = get_publisher_wallet()
+
+    sample_ddo_path = get_resource_path("ddo", "ddo_sa_sample.json")
+    asset = DDO(json_filename=sample_ddo_path)
+    asset.metadata["main"]["files"][0]["checksum"] = str(uuid.uuid4())
+    my_secret_store = "http://myownsecretstore.com"
+    auth_service = ServiceDescriptor.authorization_service_descriptor(my_secret_store)
+
+    assert ocn.assets.create(
```

Comment:
The purpose of the `owner_address` parameter is to specify an address that is different from the publisher address so that ownership can be transferred immediately after the asset is published. 

Recommended actions:

1. Duplicate this test.
2. Change the first test so that the `owner_address` parameter is different from the publisher address.
3. Keep the second test the same.

Reasoning:
This tests both the intended and unintended usage of the method.

Reply:
@DMats that is covered in integration tests, but if @mariacarmina has the time, I'd suggest we cover it here, too.

Reply:
will be addressed in #284